integration/spark/src/test/scala/org/apache/carbondata/spark/testsuite/iud/UpdateCarbonTableTestCase.scala

```diff
@@ -1178,6 +1178,15 @@ class UpdateCarbonTableTestCase extends QueryTest with BeforeAndAfterAll {
     }
   }
 
+  test("update table having alias table name - case insensitive check") {
+    sql("""drop table if exists iud.zerorows""").collect()
+    sql("""create table iud.zerorows (c1 string,c2 int,c3 string,c5 string) STORED AS carbondata""")
+    sql(s"""LOAD DATA LOCAL INPATH '$resourcesPath/IUD/dest.csv' INTO table iud.zerorows""")
+    sql("update iud.zerorows up_TAble set(up_table.c1)=('abc') where up_TABLE.c2=1")
```

Comment:
here change column name also to other case, so even that will be validated in test case

Reply:
done